04-Addressing control failure/RemediationThroughScriptsUsingUI.md

```diff
@@ -0,0 +1,32 @@
+> The Azure Tenant Security Solution (AzTS) was created by the Core Services Engineering & Operations (CSEO) division at Microsoft, to help accelerate Microsoft IT's adoption of Azure. We have shared AzTS and its documentation with the community to provide guidance for rapidly scanning, deploying and operationalizing cloud resources, across the different stages of DevOps, while maintaining controls on security and governance.
+<br>AzTS is not an official Microsoft product – rather an attempt to share Microsoft CSEO's best practices with the community.
+<br>
+
+# AutoRemediation
+This feature is not enabled by default. Default AzTS UI with AutoRemediation feature off :
+
+![UIWithAutoRediationDisabled](../Images/04_DefaultUIWithAutoRemediationDisabled.png)
+
+If you have not enabled this feature in your AzTS setup yet, please follow steps mentioned below:
```

Comment:
If you have not enabled this feature in your AzTS setup yet -> To enable this feature for your AzTS setup